js/scripts.js

```diff
@@ -0,0 +1,13 @@
+
+var withButtonClass = document.getElementsByClassName("button");
+	for (var i = 0; i < withButtonClass.length; i++) {
```

Comment:
wszystko od linijek 4 do 8 jest do wywalenia. Daj sobie console.log z `withButtonClass` i odpowiedz na pytanie czym to jest. Potem w pętli spróbuj dostać się do jego pojedynczych elementów